testing/regress/ecl/despray.ecl

```diff
@@ -274,6 +274,29 @@ c10 := CATCH(NOFOLD(p10), ONFAIL(TRANSFORM(rec,
     o10 := output(c10, {result});
 #end
 
+
+
+// This should pass based on valid relative file path and valid source address used
+DestFile11 := 'test/' + WORKUNIT + '-' + File;
```

Comment:
minor: as per previous comment, for consistency should use DestFileName here.

Reply:
Changed